canary_config.example.py

```diff
@@ -1,12 +1,14 @@
 from viam.rpc.dial import Credentials
 
-
```

Comment:
Hm... my instinct is to be nitpicky and say put this back. Per [PEP8](https://peps.python.org/pep-0008/#blank-lines), top-level paragraphs should be separated by two blank lines. Similarly, I think line 12 should be two blank lines again, and maybe even lines 5 and 9 should be two blank lines. However, that would make the file, like, half blank. So, maybe it's fine as-is.